docs/errors.rst

```diff
@@ -60,15 +60,30 @@ is no need to retry over time, as it will not become better::
             raise kopf.PermanentError("The object is not valid anymore.")
 
 
-
 Regular errors
 ==============
 
-Any other exceptions behave as either the retriable (default) or fatal,
-depending on the settings.
+Kopf assumes that any arbitrary errors
+(i.e. not `TemporaryError` and not `PermanentError`)
+are the environment issues and can self-resolve after some time.
```

Comment:
should be: "are environment issues" without "the"

Reply:
Fixed.